cars/v1/vanilla/templates/config/elasticsearch.yml

```diff
@@ -69,14 +69,14 @@ transport.port: {{transport_port}}
 # The default list of hosts is ["127.0.0.1", "[::1]"]
 #
 {%- if all_node_ips|length > 4 %}
-{%- if all_node_ips_count|default(2) > 1 %}
+  {%- if all_node_ips_count|default(2) > 1 %}
```

Comment:
Use `2` because we want `discovery.seed_hosts` and `cluster.initial_master_nodes` if a version of Rally that does not set `all_node_ips_count` is in use.